lib/hanami/helpers/form_helper/form_builder.rb

```diff
@@ -751,11 +751,42 @@ def password_field(name, attributes = {})
         #   #    <option value="it" selected="selected">Italy</option>
         #   #    <option value="us">United States</option>
         #   #  </select>
+        #
+        # @example Include blank option
+        #   <%=
+        #     # ...
+        #     values = Hash['it' => 'Italy', 'us' => 'United States']
+        #     select :stores, values, options: {include_blank: true}
+        #   %>
+        #
+        #   # Output:
+        #   #  <select name="book[store]" id="book-store">
+        #   #    <option></option>
+        #   #    <option value="it">Italy</option>
+        #   #    <option value="us">United States</option>
+        #   #  </select>
+        #
+        # @example Prompt option
+        #   <%=
+        #     # ...
+        #     values = Hash['it' => 'Italy', 'us' => 'United States']
+        #     select :stores, values, options: {prompt: 'Select a City'}
+        #   %>
+        #
+        #   # Output:
+        #   #  <select name="book[store]" id="book-store">
+        #   #    <option>Select a City</option>
+        #   #    <option value="it">Italy</option>
+        #   #    <option value="us">United States</option>
+        #   #  </select>
         def select(name, values, attributes = {})
-          options    = attributes.delete(:options) || {}
-          attributes = { name: _input_name(name), id: _input_id(name) }.merge(attributes)
+          options      = attributes.delete(:options) { {} }
+          attributes   = { name: _input_name(name), id: _input_id(name) }.merge(attributes)
+          prompt_value = options.delete(:include_blank) || options.delete(:prompt)
 
           super(attributes) do
+            option(prompt_value.is_a?(String) ? prompt_value : '') if prompt_value
```

Comment:
I don't like this solution, but when `prompt_value` is `true` `prompt_value.to_s` returns `'true'` string. If you have better solution I'll happy to use it :+1: